alphabase/statistics/regression.py

```diff
@@ -0,0 +1,235 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../../nbdev_nbs/statistics/regression.ipynb.
+
+# %% auto 0
+__all__ = ['EPSILON', 'LOESSRegression']
+
+# %% ../../nbdev_nbs/statistics/regression.ipynb 2
+import numpy as np
+
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.base import BaseEstimator, RegressorMixin
+
+from sklearn.utils.estimator_checks import check_estimator
+import matplotlib.pyplot as plt
+
+# %% ../../nbdev_nbs/statistics/regression.ipynb 3
+EPSILON = 1e-6
+
+class LOESSRegression(BaseEstimator, RegressorMixin):
+
```

Comment:
In nbdev2 (version last month), when we have doc_string below `__init__()`, nbdev2 will look for doc_string from base classses. It is annoying but we should follow its rule (or bug?). Here we can:

1. move global description of `LOESSRegression` right below the class declaration
2. keep parameters under `__init__()`